site/content/en/docs/concepts/admission.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Admission"
+date: 
+weight: 
+description: >
+  Kueue's admission process where quota reservation meets physical capacity guarantees, using ProvisioningRequest for autoscaling integration.
+---
+
+Kueue's admission process determines whether a Workload can begin execution. It involves verifying logical resource availability through quota reservation, followed by performing AdmissionChecks to validate physical resource provisioning and policy compliance. 
+
+Kueue implements this through a two-phase admission cycle: 
+
+1. **Quota Reservation:** When a user submits a Workload, it enters a LocalQueue first. This LocalQueue points to a ClusterQueue which is responsible for managing the available resources. The Kueue checks if the targeted ClusterQueue's available quota and resource flavors can accomodate requested resources (CPU, memory, GPUs, etc.). If the quota is available, the Kueue reserves resources for this Workload and prevents other Workloads from using the same resources. 
+
+2. **Admission Checks:** After the quota reservation, Kueue executes all [AdmissionChecks](/docs/concepts/admission_check) configured in the ClusterQueue, concurrently. These are the pluggable controllers that can perform validations such as policy checks, compliance, etc.
```

Comment:
```suggestion
2. **Admission Checks:** Await for [AdmissionChecks](/docs/concepts/admission_check) configured in the ClusterQueue. These can be either built-in like MultiKueue or ProvisioningRequest, or are the pluggable
controllers that can perform validations such as policy checks, compliance, etc.
```
Also, provide links to MultiKueue, ProvisioningRequest when mentioning the concepts.